sklearn/linear_model/tests/test_base.py

```diff
@@ -427,7 +434,8 @@ def test_preprocess_data_multioutput():
 
 
 @pytest.mark.parametrize("is_sparse", [False, True])
-def test_preprocess_data_weighted(is_sparse):
+def test_preprocess_data_weighted(is_sparse, global_random_seed):
+    rng = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all

Reply:
```suggestion
    rng = np.random.RandomState(global_random_seed)
```